lib/lambda/update/updatePackage.ts

```diff
@@ -0,0 +1,217 @@
+import { response } from "libs/handler-lib";
+import { APIGatewayEvent } from "aws-lambda";
+import { getPackage, getPackageChangelog } from "libs/api/package";
+import { produceMessage } from "libs/api/kafka";
+import { ItemResult } from "shared-types/opensearch/main";
+import { events } from "lib/packages/shared-types";
+import { z } from "zod";
+
+const sendDeleteMessage = async (topicName: string, packageId: string) => {
+  await produceMessage(
+    topicName,
+    packageId,
+    JSON.stringify({
+      id: packageId,
+      deleted: true,
+      isAdminChange: true,
+      adminChangeType: "delete",
+    }),
+  );
+};
+
+const sendUpdateValuesMessage = async (
+  topicName: string,
+  packageId: string,
+  updatedFields: object,
+  changeMadeText: string,
+  changeReason?: string,
+) => {
+  await produceMessage(
+    topicName,
+    packageId,
+    JSON.stringify({
+      id: packageId,
+      ...updatedFields,
+      isAdminChange: true,
+      adminChangeType: "update-values",
+      changeMade: changeMadeText,
+      changeReason,
+    }),
+  );
```

Comment:
Would it make sense to return `response`, so we can just return `sendUpdateValuesMessages` within the conditionals (e.g. `if (action === 'delete')`?

```ts
return response({
  statusCode: 200,
  body: { message: "success" },
});
```

Example returning `sendUpdateValuesMessage`:

```ts
  if (action === "delete") {
    return sendDeleteMessage(topicName, packageId);
  }
```

Reply:
Sure! I'm just moving the returned response inside the `sendUpdateValuesMessages` function right? 

For the `sendUpdateIdMessage` function, though, I don't want to return the response from `sendDeleteMessage` if we were to also move the returned response in there as well